Cargo.toml

```diff
@@ -7,9 +7,8 @@ description = "Generate string parseable in compiled binary"
 license = "MIT"
 
 [dependencies]
-proc-macro2 = "1.0"
-syn = "1.0"
-quote = "1.0"
+syn = "2.0.18"
```

Comment:
Do not enforce any requirements, use 2.0 and 1.0, like before.

Reply:
Changed to major-only requirements